### PR TITLE
Update Classifications and security vetting content

### PR DESCRIPTION
**Pull request by karlchillmaid**
This PR updates the Classifications and security vetting section of Notify's Security page.

### CODE REVIEW

File: app/templates/views/security.html [hunk 1]
```diff
@@ -77,7 +77,8 @@ <h2 class="heading-medium">Cabinet Office approval</h2>
   <p>Notify also has approval from the Office of the Government’s SIRO to host data within the EEA.</p>
 
   <h2 class="heading-medium">Classifications and security vetting</h2>
-  <p>Any information in Notify is classified as ‘OFFICIAL’ under the Government Security Classifications Policy.</p>
-  <p>All system administration staff working on Notify are cleared to Security Check (SC) level by United Kingdom Security Vetting.</p>
+  <p>You can use Notify to send messages classified as 'OFFICIAL' or 'OFFICIAL-SENSITIVE' under the <a href="https://www.gov.uk/government/publications/government-security-classifications">Government Security Classifications</a> policy.</p>
```
Comment (karlchillmaid) [inline app/templates/views/security.html]: The link to the policy gives users more information if they want it

File: app/templates/views/security.html [hunk 2]
```diff
@@ -77,7 +77,8 @@ <h2 class="heading-medium">Cabinet Office approval</h2>
...
+  <p>You must not use Notify to send '<a href="https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/special-category-data/">special category data</a>', as defined in the General Data Protection Regulation (GDPR).</p>
```
Comment (karlchillmaid) [inline app/templates/views/security.html]: Not sure if there's a precedent for linking to third party sites, but the ICO seems like a reliable source – and the full description of what 'special category data' means is useful.

File: app/templates/views/security.html [hunk 3]
```diff
@@ -77,7 +77,8 @@ <h2 class="heading-medium">Cabinet Office approval</h2>
...
+  <p>The Notify team has Security Check (SC) level clearance from <a href="https://www.gov.uk/guidance/security-vetting-and-clearance">United Kingdom Security Vetting</a> (UKSV).</p>
```
Comment (karlchillmaid) [inline app/templates/views/security.html]: 'The Notify team' is less formal (and shorter) – but is it still accurate?

**Comment by yahoopete** [comment]
Yep, it's good. Ship.it.

**Pull request merged**